misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -1,16 +1,15 @@
 package misk.redis
 
-import misk.time.FakeClock
 import okio.ByteString
-import okio.ByteString.Companion.encodeUtf8
+import java.time.Clock
 import java.time.Duration
 import java.time.Instant
 import java.util.concurrent.ConcurrentHashMap
 import javax.inject.Inject
 
 /** Mimics a Redis instance for testing. */
 class FakeRedis : Redis {
-  @Inject lateinit var clock: FakeClock
+  @Inject lateinit var clock: Clock
```

Comment:
Because this file is in implementation classes we have no access to FakeClock. In tests, this should bind to one.